### PR TITLE
AAP-4587 Add info for connecting hub to existing SSO

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Documentation for connecting Automation Hub to an existing Red Hat SSO

Uses SSO inventory variables added in #916 

Affects `/titles/hub/installi`

### CODE REVIEW

File: downstream/titles/hub/getting-started/master.adoc [hunk 1]
```diff
@@ -29,3 +29,5 @@ include::automation-hub/proc-configure-automation-hub-server.adoc[leveloffset=+1
 You have now configured {HubName} as your default server and can proceed to download and install supported collections.
 
 For more information on server list configuration options and using Ansible Galaxy as an Ansible content source, see the https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#configuring-the-ansible-galaxy-client[Ansible Galaxy User Guide].
+
```
Comment (ianf77) [inline downstream/titles/hub/getting-started/master.adoc:32]: On line 31 you can  use {Galaxy} for Ansible Galaxy

**Review by ianf77 — COMMENTED** [review]
One comment

**Review by ianf77 — APPROVED** [review]
LGTM

File: downstream/titles/hub/getting-started/master.adoc [hunk 2]
```diff
@@ -12,7 +12,7 @@ include::attributes/attributes.adoc[]
 
 Red Hat Ansible {HubName} provides a place for Red Hat subscribers to quickly find and use content that is supported by Red Hat and our technology partners to deliver additional reassurance for the most demanding environments.
 
-The Ansible Galaxy client, `ansible-galaxy`, manages roles and collections from the command line. To ensure that the `ansible-galaxy` client uses certified, supported Ansible collections whenever possible, you should update your `ansible.cfg` file to use Red Hat {HubName} as your primary source of Ansible collections.
+The {Galaxy} client, `ansible-galaxy`, manages roles and collections from the command line. To ensure that the `ansible-galaxy` client uses certified, supported Ansible collections whenever possible, you should update your `ansible.cfg` file to use Red Hat {HubName} as your primary source of Ansible collections.
```
Comment (ianf77) [inline downstream/titles/hub/getting-started/master.adoc:15]: possible, update your

**Review by ianf77 — COMMENTED** [review]
Couple of comments, you can ignore them if you want.

File: downstream/modules/automation-hub/proc-connect-hub-existing-sso-aap.adoc [hunk 3]
```diff
@@ -0,0 +1,26 @@
+[id="proc-connect-hub-existing-sso-aap"]
+= Configuring {PlatformNameShort} managed {RHSSO} inventory file variables
+
+To connect {HubName} to an existing {RHSSO} installation that is managed by {PlatformNameShort}, configure the following variables in the `inventory` file.
+
+
+[cols="30%,70%",options="header"]
+|====
+| *Variable* | *Description*
```
[the remark this answers is not in the document]
Comment (ariordan-redhat) [reply downstream/modules/automation-hub/proc-connect-hub-existing-sso-aap.adoc:9]: Variables moved to inventory files in PR #916 - descriptions added here.

**Review by ianf77 — APPROVED** [review]
LGTM

File: downstream/modules/automation-hub/proc-run-setup-script-hub.adoc [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[id="run-setup-script-hub"]
+
+= Running the setup script
+
+You can run the setup script once you finish updated the `inventory` file with required parameters for installing your Private Automation Hub.
```
Comment (ianf77) [inline downstream/modules/automation-hub/proc-run-setup-script-hub.adoc:5]: when you finish updating the `inventory file` with the required parameters for installing your {PrivateHubName}.

**Review by ianf77 — COMMENTED** [review]
A couple of comments.

**Comment by ariordan-redhat** [comment]
@simaishi I made changes to the PR and updated the preview.

**Comment by ariordan-redhat** [comment]
@simaishi I updated the preview with the most recent changes.

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**